.vib/minio/cypress/cypress/integration/minio_spec.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
```

Comment:
You may need to rebase your PR with master, as these changes were already integrated.

Reply:
OK, noted!